src/ert/ensemble_evaluator/snapshot.py

```diff
@@ -219,38 +214,39 @@ def get_forward_model_status_for_all_reals(
 
     @property
     def reals(self) -> Mapping[str, "RealizationSnapshot"]:
-        return {
-            real_id: RealizationSnapshot(**real_data)
-            for real_id, real_data in self._realization_states.items()
-        }
+        return self._realization_states
 
-    def get_forward_models_for_real(self, real_id: str) -> Dict[str, "ForwardModel"]:
+    def get_forward_models_for_real(
+        self, real_id: RealId
+    ) -> Dict[FmStepId, "ForwardModel"]:
         return {
             fm_idx[1]: forward_model_data.copy()
             for fm_idx, forward_model_data in self._forward_model_states.items()
             if fm_idx[0] == real_id
         }
 
-    def get_real(self, real_id: str) -> "RealizationSnapshot":
-        return RealizationSnapshot(**self._realization_states[real_id])
+    def get_real(self, real_id: RealId) -> "RealizationSnapshot":
+        return self._realization_states[real_id]
 
-    def get_job(self, real_id: str, forward_model_id: str) -> "ForwardModel":
+    def get_job(self, real_id: RealId, forward_model_id: FmStepId) -> "ForwardModel":
         return self._forward_model_states[(real_id, forward_model_id)].copy()
 
     def get_successful_realizations(self) -> typing.List[int]:
         return [
             int(real_idx)
             for real_idx, real_data in self._realization_states.items()
-            if real_data[ids.STATUS] == state.REALIZATION_STATE_FINISHED
+            if real_data["status"] == state.REALIZATION_STATE_FINISHED
```

Comment:
why changing from `ids.STATUS` TO `"status"`?

Reply:
I think we should move away from using `ids` altogether in most of the codebase. The benefits are already given to us for free with typed dicts.